plugins/carbon_black_cloud/help.md

```diff
@@ -298,6 +298,7 @@ Example output:
 
 # Version History
 
+* 2.0.1 - Allows hostnames provided to be case insensitive | Fix bug where error is raised if endpoint was not found in `get_agent` method | To add escaping of special characters in hostnames when preforming hostname searches to Carbon Black
```

Comment:
typo in performing (preforming). Can you add in details of where the hostnames provided can be case sensitive?